components/support/migration/src/main/java/mozilla/components/support/migration/FennecLoginsMigration.kt

```diff
@@ -155,26 +155,13 @@ internal object FennecLoginsMigration {
             return Result.Failure(LoginMigrationException(LoginsMigrationResult.Failure.GetLoginsThrew(e)))
         }
 
-        val migrationMetrics = try {
-            loginsStorage.importLoginsAsync(fennecRecords.records)
-        } catch (e: Exception) {
-            return Result.Failure(LoginMigrationException(LoginsMigrationResult.Failure.RustImportThrew(e)))
-        }
-
-        // TODO process login migration metrics properly:
-        // See https://github.com/mozilla/application-services/commit/d1a12d98a8c428567a3fff9e4333869b980952a8
-        val failedToImport = try {
-            migrationMetrics.getInt("num_failed")
-        } catch (e: Exception) {
-            crashReporter.submitCaughtException(e)
-            0
-        }
+        loginsStorage.importLoginsAsync(fennecRecords.records)
```

Comment:
Is the Fennec migration still supported?  Maybe it would be better to just remove this code.

Reply:
Yes, I think we remove the Fennec migration code path https://github.com/mozilla-mobile/fenix/pull/23651

Reply:
I made #12491 for removing the a-c migration code entirely.  I split it out because it seemed a bit more risky than this change.